aiida/cmdline/commands/cmd_export.py

```diff
@@ -89,10 +89,13 @@ def create(
     create_backward, return_backward, call_calc_backward, call_work_backward, include_comments, include_logs
 ):
     """
-    Export parts of the AiiDA database to file for sharing.
+    Export subsets of the provenance graph to file for sharing.
 
-    Various entities can be exported, such as Codes, Computers, Groups, Nodes,
-    Comments, Logs, ...
+    Besides Nodes of the provenance graph, you can export Groups, Codes, Computers, Comments, Logs and more.
```

Comment:
"and more" like what? I think we have listed everything there.
Also technically "Codes" are nodes, but I can see how it makes sense to list them invidivually given that there is a separate flag, so fine to keep.

Reply:
Ok, will delete (this was just a translation of the ...)